Source/WebCore/platform/audio/ios/MediaSessionManagerIOS.mm

```diff
@@ -152,8 +130,6 @@
     session.setShouldPlayToPlaybackTarget(playbackTargetSupportsAirPlayVideo);
 #endif
 
-    providePresentingApplicationPIDIfNecessary(session.presentingApplicationPID());
```

Comment:
In the case of a muted video, we would no longer set the presenting application PID since the remote audio session would not be activated. I don't think there is any user impact though.